x-pack/plugins/apm/public/components/app/metrics/static_dashboard/dashboards/dashboard_catalog.ts

```diff
@@ -7,6 +7,7 @@
 
 export const AGENT_NAME_DASHBOARD_FILE_MAPPING: Record<string, string> = {
   nodejs: 'nodejs',
+  'opentelemetry/nodejs': 'opentelemetry-nodejs',
```

Comment:
nit: I would use `opentelemetry_nodejs` (underscore) similar to the filename for consistency.

Reply:
done!